html/elements/meta.json

```diff
@@ -675,10 +675,12 @@
                   "version_added": false
                 },
                 "safari": {
-                  "version_added": false
+                  "version_added": "15",
+                  "notes": "Safari 15 beta supports a media attribute that recognizes (prefers-color-scheme: light) or (prefers-color-scheme: dark)."
                 },
                 "safari_ios": {
-                  "version_added": false
+                  "version_added": "15",
+                  "notes": "Safari 15 beta supports a media attribute that recognizes (prefers-color-scheme: light) or (prefers-color-scheme: dark) on iPadOS 15."
```

Comment:
If this is referring to the new "theme-color" meta tag support, it isn't iPad OS only, it's also support on iOS.

Source: https://developer.apple.com/documentation/safari-release-notes/safari-15-beta-release-notes#:~:text=Added%20support%20for%20the%20theme-color%20meta%20tag%20to%20change%20the%20tab%20bar%20background%20and%20over-scroll%20area%20in%20macOS%20and%20iPadOS%2C%20and%20the%20status%20bar%20in%20iOS

Reply:
I verified that this is indeed correct.